eos_data_distribution/ndn/http.py

```diff
@@ -82,7 +90,9 @@ def __init__(self, url, session=None, chunk_size=chunks.CHUNK_SIZE):
 
         # XXX -- this is a bit ugly that we're making an HTTP request
         # in the constructor here...
-        self._size = get_content_size(self._session, self.url)
+        self._headers = get_headers(self._session, self.url)
```

Comment:
`get_headers` is a bit generic of a function name here, and `get` usually doesn't imply network activity. A different name would be better here...